typescript/organisations.d.ts

```diff
@@ -84,6 +84,7 @@ export interface NewTier {
     allowFormsAppMenu: boolean
     allowApprovalsSolution?: boolean
     allowVolunteersSolution?: boolean
+    allowSubmissionTitleSearch: boolean
```

Comment:
```suggestion
    allowSubmissionTitleSearch?: boolean
```

to a avoid a breaking change to the API